### PR TITLE
re-template and fix node decommission

### PR DESCRIPTION
**Pull request by be99inner**
- re structure for default
- fix node decommission

### CODE REVIEW

**Review by nbzero — APPROVED** [review]
LGTM

**Review by nbzero — APPROVED** [review]
LGTM

File: tasks/cassandra_post.yml [hunk 1]
```diff
@@ -1,48 +1,69 @@
...
-    backup: yes
   register: cassandra_init_config_reg
   tags: cassandra-config
 
 - block: # Clean up system configuration
-  - name: Stop cassandra for Reload configuration
-    systemd:
-      name: cassandra
-      state: stopped
   - find:
```
Comment (winggundamth) [inline tasks/cassandra_post.yml]: Put - name: to descript task

**Review by winggundamth — CHANGES_REQUESTED** [review]
Please check

**Review by winggundamth — APPROVED** [review]
LGTM

**Pull request merged**